raw_data.php

```diff
@@ -693,6 +703,9 @@ function get_stops($swLat, $swLng, $neLat, $neLng, $tstamp = 0, $oSwLat = 0, $oS
 
 function get_gyms($swLat, $swLng, $neLat, $neLng, $tstamp = 0, $oSwLat = 0, $oSwLng = 0, $oNeLat = 0, $oNeLng = 0)
 {
+    if (!is_null($adapter)) {
+      return $adapter->get_gyms($swLat, $swLng, $neLat, $neLng, $tstamp, $oSwLat, $oSwLng, $oNeLat, $oNeLng);
```

Comment:
When $adapter is defined, you can use it to get_gyms from adapter. Notice that, there is no `monkey` around here.